python/mxnet/model.py

```diff
@@ -789,6 +819,117 @@ def fit(self, X, y=None, eval_data=None, eval_metric='acc',
                             sym_gen=self.sym_gen)
 
 
+    def profiling(self, X, y=None, eval_data=None, eval_metric='acc',
```

Comment:
As you can see in [example/profiler/profiler_modelfit.py](https://github.com/ZihengJiang/mxnet/blob/master/example/profiler/profiler_modelfit.py#L83), if we want to profile the efficiency of model.fit interface, we either add some new arguments (like model.fit(profiling=True, begin_profiling_iteration=8,...) in the original model.fit interface or create a separate function which has all feature of model.fit like model.profiling at here. Which one do you think is better?